apps/app/eslint.config.cjs

```diff
@@ -1,42 +1,48 @@
-const { FlatCompat } = require('@eslint/eslintrc');
+const nx = require('@nx/eslint-plugin');
 const baseConfig = require('../../eslint.config.cjs');
-const js = require('@eslint/js');
-
-const compat = new FlatCompat({
-	baseDirectory: __dirname,
-	recommendedConfig: js.configs.recommended,
-});
 
 module.exports = [
 	...baseConfig,
-	...compat
-		.config({ extends: ['plugin:@nx/angular', 'plugin:@angular-eslint/template/process-inline-templates'] })
-		.map((config) => ({
-			...config,
-			files: ['**/*.ts'],
-			rules: {
-				'@angular-eslint/no-host-metadata-property': 0,
```

Comment:
Do we still need this?

Reply:
Just to clarify, are you referring to `'@angular-eslint/no-host-metadata-property'` or `@angular-eslint/directive-selector`?

GitHub is showing me the comment on both lines depending on where i view it 😂

![image](https://github.com/user-attachments/assets/0d159f8c-8c5c-4fd8-a035-461173f8c51b)

![image](https://github.com/user-attachments/assets/54a625b7-1d86-4220-b18b-adf78109b605)

Reply:
I was referring to: `@angular-eslint/no-host-metadata-property`

Reply:
Ah right, this is now the default behaviour and the rule is deprecated, so removing it as we inherit this from the recommended rules.